WordPress/src/main/java/org/wordpress/android/ui/reader/services/ReaderPostService.java

```diff
@@ -195,10 +196,17 @@ public void onErrorResponse(VolleyError volleyError) {
             }
         };
 
-        WordPress.getRestClientUtilsV1_1().get(sb.toString(), null, null, listener, errorListener);
+        // TODO: as of 09-July-2015 there is no v1.2 endpoint for Freshly Pressed, so we have to
```

Comment:
We should probably file an issue to remind us to use the v1.2 endpoint exclusively when it supports FP.